cmd/commands.go

```diff
@@ -133,6 +134,63 @@ func New() *cli.Command {
 	return cmd
 }
 
+type BearerAuthentication struct {
+	Token string
+}
+
+func (a *BearerAuthentication) GetRequestMetadata(context.Context, ...string) (map[string]string, error) {
+	return map[string]string{
+		"Authorization": fmt.Sprintf("Bearer %s", a.Token),
+	}, nil
+}
+
+func (a *BearerAuthentication) RequireTransportSecurity() bool {
+	return false
+}
+
+type BasicAuthentication struct {
+	Token string
+}
+
+func (a *BasicAuthentication) GetRequestMetadata(context.Context, ...string) (map[string]string, error) {
+	return map[string]string{
+		"Authorization": fmt.Sprintf("Basic %s", a.Token),
+	}, nil
+}
+
+func (a *BasicAuthentication) RequireTransportSecurity() bool {
+	return false
+}
+
+func initClientConnection(l log.Logger, serverHost string, requestTimeout time.Duration) (
```

Comment:
will it be a good idea to move all GRPC client-related code to a separate file?

Reply:
yeah, that could be a good idea. will try to check it and update it accordingly.

Reply:
it's addressed